modules/nodes-rwn_con_adding-remote-worker-nodes.adoc

```diff
@@ -0,0 +1,19 @@
+// This module is included in the following assemblies: 
```

Comment:
I would rename this module so that it uses - and not _ . I imagine you were trying to follow [this](https://redhat-documentation.github.io/modular-docs/#module_anchor-and-file-names-concept) but I think most files in the repo use - and _ are reserved for folders.